Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/AssetFolderThumbnailView.cpp

```diff
@@ -948,6 +948,11 @@ namespace AzQtComponents
         m_showSearchResultsMode = searchMode;
     }
 
+    void AssetFolderThumbnailView::HideProductAssets(bool checked)
```

Comment:
Mega ultra nitpick: it's more common to form bool variable names like questions, so `isHidden` instead of `checked`.